tls_assets.go

```diff
@@ -76,6 +78,13 @@ var ClusterComponents = []ClusterComponent{
 	ServiceAccountComponent,
 }
 
+// MonitoringComponents is a slice enumerating all the components that make up
+// monitoring.
+var MonitoringComponents = []ClusterComponent{
+	PrometheusComponent,
```

Comment:
This is probably named badly at this point. We should be using separate certificates for each monitoring components - this is for the API server. Learning as we go along. Planning on cleaning this up shortly.

Reply:
Works for me. Maybe just add some comments where you can explain things.